### ISSUE TITLE
Clicking login button results in a 404

### ISSUE THREAD

**Issue by losttheplot** [todstoychev/Laravel5Starter#28]
I have traced the cause to line 41 of resources/views/navigation.blade.php which is missing a slash before users, thus:

```
<a href="{{ url(\Illuminate\Support\Facades\App::getLocale() . 'users/login') }}">
should be
<a href="{{ url(\Illuminate\Support\Facades\App::getLocale() . '/users/login') }}">
```

**Comment by todstoychev** [comment]
Thanx for the response! I will fix it very soon.

**Comment by todstoychev** [comment]
This should be fixed by release v0.9.3. Thanx for the response.

**Comment by todstoychev** [comment]
Since this one is fixed from 0.9.3 it will be closed.
